docs/osctl/osctl_cluster_create.md

```diff
@@ -16,8 +16,8 @@ osctl cluster create [flags]
 ```
       --cpus string                 the share of CPUs as fraction (each container) (default "1.5")
   -h, --help                        help for create
-      --image string                the image to use (default "docker.io/autonomy/talos:v0.3.0-alpha.10-51-g3a93e65b-dirty")
-      --kubernetes-version string   desired kubernetes version to run (default "1.16.2")
+      --image string                the image to use (default "docker.io/autonomy/talos:v0.3.0-alpha.10-58-g5fb12f73-dirty")
```

Comment:
Probably need to find a more better way to do the versioning/tagging here for doc updates